ui/client/components/SpinnerWrapper.tsx

```diff
@@ -7,7 +7,7 @@ type Props = {
 }
 
 function SpinnerWrapper({isReady, children}: PropsWithChildren<Props>) {
-  return isReady ? children : <LoaderSpinner show={true}/>
+  return isReady ? <>{children}</> : <LoaderSpinner show={true}/>
```

Comment:
Why it is needed? This is some kind of "hack" to ensure that return type will be dom node? Doesn't it change rendering?